crates/bevy_render/src/render_phase/mod.rs

```diff
@@ -76,7 +106,94 @@ impl<I: BatchedPhaseItem> RenderPhase<I> {
     }
 }
 
-/// This system sorts all [`RenderPhases`](RenderPhase) for the [`PhaseItem`] type.
+/// An item (entity) which will be drawn to a texture or the screen, as part of a [`RenderPhase`].
+///
+/// A phase item has to be queued up for rendering during the
+/// [`RenderStage::Queue`](crate::RenderStage::Queue).
+/// Afterwards it will be sorted and rendered automatically in the
+/// [`RenderStage::PhaseSort`](crate::RenderStage::PhaseSort) and
+/// [`RenderStage::Render`](crate::RenderStage::Render), respectively.
+pub trait PhaseItem: Sized + Send + Sync + 'static {
```

Comment:
Moving this is a breaking change. Are you sure we should be doing this in a docs PR?

Reply:
Is it? I thought that `PhaseItem` is imported using 
```rust
use bevy::render::render_phase::PhaseItem;
```
anyway. Since all types of this module are exported in `render_phase` anyway, I thought that this move does not break anything.

If you want I can revert this or even split it into its own PR.
My goal with this PR was to clarify the contents of this module. I believe that putting `RenderPhase` and `PhaseItem` into the same file, as well as, `Draw` function and `RenderCommand` makes the most sense.

Reply:
Just checked the module imports and this does seem like it is clustered at the higher module. This is good then.